sgl-kernel/tests/test_moe_fused_gate.py

```diff
@@ -43,8 +46,15 @@ def test_moe_fused_gate_combined(seq_length, dtype, params):
         num_expert_group=num_expert_group,
         topk_group=topk_group,
         compiled=False,
+        n_share_experts_fusion=n_share_experts_fusion,
     )
 
+    # When n_share_experts_fusion > 0, ignore the comparison of the last topk dimension
+    if n_share_experts_fusion > 0:
```

Comment:
would be great if we add a few lines to assert that `indices[:, -1]` is within the correct value ranges

and if we use round-robin, we can further assert it like `it == [64,65,66,67,64,65,66,67,...]` etc

Reply:
done.